README.md

```diff
@@ -1,6 +1,25 @@
-# Guiguts-2.0-tkinter
+# Guiguts
+
+Guiguts - application to support creation of books for PG
+
+## Licensing
```

Comment:
I'd be inclined to throw this at the bottom of the README rather than at the top since it's important to have but not generally useful to most people who open the README.

Reply:
Should that be "ebooks" (or some variation on the word) instead of "books"?

Reply:
Yes to both